src/main/java/core/basesyntax/io/CsvWriter.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.io;
+
+import core.basesyntax.io.exception.CsvException;
+import core.basesyntax.model.Report;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvWriter {
+    private static final String DELIMITER = ",";
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final String REPORT_FORMAT = "supply%s%d%sbuy%s%d%sresult%s%d";
```

Comment:
The 'REPORT_FORMAT' string does not need to include '%s' for the delimiter and new line constants. The format placeholders should only be used for the dynamic parts of the string, such as the report data fields.